book/routing.rst

```diff
@@ -811,14 +818,21 @@ Path     Parameters
 .. index::
    single: Routing; Method requirement
 
-Adding HTTP Method Requirements
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+HTTP Method Requirements
+~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Let's keep this - it doesn't hurt, and we won't have to add a manual anchor to *not* break the links